index.js

```diff
@@ -4,7 +4,45 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    let arr = process.argv.slice(2);
+    let sum = Number(arr[1]);
+
+    if (arr.length === 0) {
+        return "No operation provided...";
+    } else if (arr[0] !== 'minus' && arr[0] !== 'plus') {
+        return "Invaild operation: ${arr[0]}";
+    } else if (arr[1] === undefined) {
+        return ("No numbers provided...");
+    }
+    
+    if (arr[0] === 'plus') {
+        for (let i = 1; i < arr.length - 1; i++) {
```

Comment:
Since you already initialized the variable `sum` with `arr[1]` , you can initialize `i` with `let i=2` or another approach can be to initialize `sum = 0` and add the numbers or subtract as you loop.